python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -1170,13 +1205,21 @@ def buildDifferenceImage(self, skyInfo, tempExpRefList, imageScalerList, weightL
         @param bgModelList: List of background models from background matching
         @return Difference image of unclipped and clipped coadd wrapped in an Exposure
         """
+        # Clone and upcast self.config because current self.config is frozen
+        config = AssembleCoaddConfig()
+        # getattr necessary because subtasks do not survive Config.toDict()
+        configIntersection = {k: getattr(self.config, k)
+                              for k, v in self.config.toDict().items() if (k in config.keys())}
+        config.update(**configIntersection)
+
         # Build the unclipped coadd
-        coaddMean = AssembleCoaddTask.assemble(self, skyInfo, tempExpRefList, imageScalerList, weightList,
-                                               bgModelList, doClip=False)
+        config.statistic = 'MEAN'
```

Comment:
Haven't you already forced this to be 'MEAN'?  Maybe it is just here for clarity?